src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionpostponementrequest/ActionPostponementRequestAboutToSubmitHandler.java

```diff
@@ -174,7 +196,19 @@ private void clearTransientFields(SscsCaseData caseData) {
 
         YesNo unprocessedPostponementRequest = caseData.getPostponementRequest().getUnprocessedPostponementRequest();
         caseData.setPostponementRequest(PostponementRequest.builder()
-            .unprocessedPostponementRequest(unprocessedPostponementRequest)
-            .build());
+                .unprocessedPostponementRequest(unprocessedPostponementRequest)
+                .actionPostponementRequestSelected(caseData.getPostponementRequest().getActionPostponementRequestSelected())
+                .build());
```

Comment:
are there any other fields in the postponement request that we're clearing here? because for unprocessedPostponementRequest and actionPostponementRequestSelected, we're just setting it to what it was before